course-details.md

```diff
@@ -1,13 +1,28 @@
 This course will dive into how you can get your best work done by identifying when and how to request a review, how to perform a review for someone else's pull request, and other awesome collaboration methods.
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub—and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
 
-In this course, you’ll learn how to:
+## What you'll learn
 
+We'll answer common questions like:
+- What is a pull request?
+- How do you comment on a pull request?
+- How do you review a pull request?
+- How do you request a pull request review?
+- How do you accept changes introduced by a pull request review?
+
+And when you're done you'll be able to:
+- Create a pull request
 - Comment on pull requests
 - Leave reviews on pull requests
-- See what’s expected when reviewing pull requests
+- Know what’s expected when reviewing pull requests
 - Request pull request reviews
 - Accept changes introduced by a pull request review
 
-This course has a dedicated message board on the [GitHub Community Forum]({{ communityBoard }}). Create a post to start a conversation, discuss this course with GitHub Trainers and participants, or troubleshoot any issues you encounter. 
+
+## Prerequisites
+None. This course is a great introduction to using pull requests.
+
```

Comment:
For the "Projects used" section, I think we can link to the source code for the Tetris game: https://github.com/jakesgordon/javascript-tetris